distribution/helm/vector-shared/templates/_labels.tpl

```diff
@@ -9,7 +9,12 @@ helm.sh/chart: {{ include "libvector.chart" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
+app.kubernetes.io/component: logging
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/part-of: {{ include "libvector.name" . }}
+{{- if .Values.customLabels }}
```

Comment:
We might want to add the option for `global` labels that would get passed to both Agents and Aggregators when using the "vector" chart.

Reply:
I do think it would be good to be able to set this with a `.Values.global.commonLabels` as well - it shouldn't be difficult to include either